requirements.txt

```diff
@@ -0,0 +1,5 @@
+numpy>=1.7
+pandas>=0.15
+python-dateutil>=2.4.2
+pytz>=2015.4
+six>=1.9.0
```

Comment:
Yes. Not completely sure about six but the other 2 (pytz and python-dateutil) are documented [here](http://pandas.pydata.org/pandas-docs/stable/install.html#dependencies). Having said that, I generated the requirements file mechanically with pip freeze and we probably could do without and let pip transitively install second order dependencies. This way, no one needs to maintain those if for whatever reason pandas does not need them anymore. I'll provide a patch.